client/bindings/pillarEditor.js

```diff
@@ -17,12 +17,17 @@ define(
             },
             update: function(element, valueAccessor, allBindings, viewModel, bindingContext) {
                 var editing = ko.unwrap(valueAccessor());
-                var pillarModel = bindingContext.$parents[2];
-
+                if (bindingContext.$parents[2].constructor.name === 'pillarModel') {
```

Comment:
this block and the block at line 70 seem identical. should you put it in a function?